airbyte_cdk/connector_builder/test_reader/reader.py

```diff
@@ -238,7 +243,11 @@ def _check_record_limit(self, record_limit: Optional[int] = None) -> int:
 
         return record_limit
 
-    def _categorise_groups(self, message_groups: MESSAGE_GROUPS) -> GROUPED_MESSAGES:
+    def _categorise_groups(
+        self,
+        message_groups: MESSAGE_GROUPS,
+        deprecation_warnings: Optional[List[Any]] = None,
```

Comment:
nit: it seems like the deprecation warnings are not necessary for categorizing groups. Should we just do `log_messages.append(deprecation_warnings)` from `run_test_read`?

Reply:
yeap!